bezier-rs/docs/interactive-docs/src/components/BezierDrawing.ts

```diff
@@ -105,7 +113,51 @@ class BezierDrawing {
 			this.options = options;
 		}
 		this.clearFigure();
-		drawBezier(this.ctx, this.points, this.dragIndex);
+
+		// For the create through points cases, we store a bezier where the handle is actually the point that the curve should pass through
+		// This is so that we can re-use the drag and drop logic, while simply drawing the desired bezier instead
+		const actualBezierPointLength = this.bezier.get_points().length;
+		let pointsToDraw = this.points;
+
+		const styleConfig = {
+			handleLineStrokeColor: COLORS.INTERACTIVE.STROKE_2,
+		};
+		let dragIndex = this.dragIndex;
+		if (this.createThroughPoints) {
+			if (actualBezierPointLength === 3) {
+				pointsToDraw = WasmBezier.quadratic_through_points(
```

Comment:
If you move these two arguments into variables defined above, can you get this to format itself onto one line?